docs/get-started/using-with-react.md

```diff
@@ -32,9 +30,33 @@ const data = [
 ];
 
 export default class App extends Component {
-
   render() {
+    return (
+      <DeckGL {...viewport} layers={[
+        new LineLayer({id: 'line-layer', data})
+      ]} />
+    );
+  }
+}
+
+```
+
+## Adding a Base Map
 
+An important companion to deck.gl is `react-map-gl`. It can provide both a base map as well as event handling for deck.gl.
+
+```jsx
+/// app.js
+import MapGL from 'react-map-gl';
+
+// Set your mapbox access token here
+const MAPBOX_ACCESS_TOKEN = 'MAPBOX_ACCESS_TOKEN';
+
+... // as in previous example
```

Comment:
not sure if these two comments are easily understandable / necessary

Reply:
Do you suggest we duplicate the entire thing?

Reply:
Either duplicate content, or omit obvious things like the viewport object that have already been explained and the react/deck.gl imports. Also maybe change the default mapbox token string to something like `pk.`, so it makes it more obvious. But that's just a thought, don't feel pressure to change it